README.md

```diff
@@ -1,4 +1,4 @@
-# `kcp` is a minimal Kubernetes API server
+# `kcp` is a minimal Kubernetes API server (and more!)
```

Comment:
Is the expectation that we didn't change this to better blurb what we are because we're waiting for p3 to get there?

Reply:
> kcp is a control plane for applications running across multiple Kubernetes clusters

Might be more accurate at this point.

I don't know that the "how minimal" is really helping anymore - we're not minimal, we're focused on solving the two problems described below.

Reply:
I wanted to come back to this. I agree w/your comments. I mostly was writing up something quickly before I had to step out for a little while 😄